recirq/qml_lfe/learn_dynamics_C.py

```diff
@@ -0,0 +1,167 @@
+import os
```

Comment:
Missing copyright.  Also, we should add a module level docstring to explain how to run this, the flags that can be used, and where/what this will output.

Reply:
Added. `python3 learn_dynamics_c.py --help` is has now got lots of info.